README.md

```diff
@@ -191,14 +211,18 @@ In short, for WPF apps, a solution based on static XAML views is currently the w
 
 #### Do I have to use the project structure outlined above?
 
-Not at all. The above example, as well as the samples, keep everything in a single project for simplicity (the samples have the XAML definitions in separate projects for technical reasons). For more complex apps, you might want to consider a more clear separation of UI and core logic. An example would be the following structure:
+Not at all. The above example, as well as the samples, keep all non-UI code in a single project for simplicity, and all the XAML in a C# project for better tooling.
+
+An alternative with a clearer separation of UI and core logic can be implemented by splitting the F# project into two projects:
```

Comment:
FWIW, I use three projects here:
1. `1_Pure`.  It has no dependencies (other than `FSharp.Core`) and contains types Model, Msg, Cmd (and maybe OutMsg and OutCmd) and functions/data init and update.  Those command types are part of the CmdMsg pattern.
2. `2_Impure`.  It has a dependency on `Elmish` (and others like `Serilog`) and maps my Cmd types to Elmish.Cmd types via a function called `bindCmd`.
3. `3_Bindings`.  It has a dependency  on `Elimish.WPF` (and others like `SkiaSharp`) and defines the bindings and design-time view models.

Then some other projects:
0. `0_Library` containing no dependencies, pure code, and project independent code.  Mostly code to support functional programming, but also some code for mathematics (such at geometery and linear algebra) and improving the .NET or `FSharp.Core` API.
4. `4_Views` containing XAML and a few C# classes.
5. `5_Testing.UnitTests`
6. `6_Testing.IntegrationTests`

Reply:
Sure. My intention is just to "whet their appetite" by suggesting that other project structures are possible, e.g. with the aim of getting a clearer separation of UI and core logic, or with an alternative entry point. I think users, when they are ready for it, are able to make the decision to split the projects up further or make adjustments. But if you feel the readme should be modified, let me know.

Reply:
No changes necessary.  Just sharing some thoughts that I had while reading that part.